otter/integration/lib/timeout.py

```diff
@@ -0,0 +1,4 @@
+class TimeoutError(Exception):
```

Comment:
Probably this can be removed, since the `TimedOutError` in `deferredutils` is being used now.